tests/sentry/issues/auto_source_code_config/test_process_event.py

```diff
@@ -48,37 +71,36 @@ def create_event(self, frames: Sequence[Mapping[str, str | bool]], platform: str
         # XXX: In the future fix store_event to return the correct type
         return cast(GroupEvent, self.store_event(data=test_data, project_id=self.project.id))
 
-    def _get_trees_for_org(self, files: Sequence[str]) -> dict[str, RepoTree]:
-        return {"test-org/repo": RepoTree(RepoAndBranch("test-org/repo", "master"), files)}
-
-    def _repo_tree_files(self, files: Sequence[str]) -> list[dict[str, Any]]:
-        return [{"path": file, "type": "blob"} for file in files]
-
-    def _repo_info(self) -> dict[str, str]:
-        return {"full_name": "test-org/repo", "default_branch": "master"}
-
     def _process_and_assert_code_mapping(
         self,
         *,  # Force keyword arguments
-        repo_files: Sequence[str],
+        repo_trees: Mapping[str, Sequence[str]],
```

Comment:
A repo tree represents each repository and all files in it.

Tests can now do this:
```python
self._process_and_assert_code_mapping(
    repo_trees={REPO1: ["src/foo/bar.py"], REPO2: ["app/baz/qux.py"],
    ...
)
```